src/lib/pipeline_configuration.py

```diff
@@ -207,9 +208,30 @@ class PipelineConfiguration(object):
                     run_id_field="organizations_run_id",
                     analysis_file_key="organizations_",
                     cleaner=None,
-                    code_scheme=CodeSchemes.ORGANIZATIONS)
+                    code_scheme=CodeSchemes.ORGANIZATIONS),
     ]
-    
+
+    LOCATION_CODING_PLANS = [
+
+        CodingPlan(raw_field="constituency_raw",
```

Comment:
In hindsight this is arguably better called "location_raw".

Reply:
Don't have to change that in this PR if it would cause too much chaos now

Reply:
I agree too location_raw is more descriptive. However due to the project current timelines I opted to leave it as is for now.